include/HistorizedMatcher.h

```diff
@@ -0,0 +1,129 @@
+// SPDX-FileCopyrightText: Deutsches Elektronen-Synchrotron DESY, MSK, ChimeraTK Project <chimeratk-support@desy.de>
```

Comment:
Either place the file into a subdirectory, or change its name to something like `DataConsistencyGroupHistorizedMatcher`. Yes it's long, but nobody is using it directly.